pom.xml

```diff
@@ -3229,17 +3229,6 @@
           <artifactSet>
             <includes>
               <include>org.spark-project.spark:unused</include>
-              <include>org.eclipse.jetty:jetty-io</include>
```

Comment:
if we move this part to `core/pom.xml`, also need remove the `relocation` of jetty from parent `pom.xml`?

Reply:
Not exactly, jetty classes should be included into `core` only, but relocation is required for all modules, otherwise, other modules like `sql` reference the vanilla jetty classes but `core` ships relocated jetty classes.

Reply:
got it